admin/admin-model.php

```diff
@@ -454,6 +454,18 @@ public function update_translations( $old_slug, $new_slug = '' ) {
 			foreach ( $terms as $term ) {
 				$term_ids[ $term->taxonomy ][] = $term->term_id;
 				$tr = maybe_unserialize( $term->description );
+
+				/**
+				 * Filters the post translations group.
+				 *
+				 * @since 3.2
+				 *
+				 * @param array  $tr       The post translations group.
+				 * @param string $old_slug The old language slug.
+				 * @param string $new_slug The new language slug.
+				 */
+				$tr = apply_filters( 'update_post_translations_group', $tr, $old_slug, $new_slug );
```

Comment:
Knowing that `$tr` is used in the preparation SQL query here : https://github.com/polylang/polylang/blob/a754f33dceb84b55cc451e6f2462e8b9ba4b396a/admin/admin-model.php#L470
I suggest to apply the filter before.

Reply:
>Although, this is not important for our current case, shouldn't we add a 4th parameter to pass the taxonomy.

Agree. Adding `$term` would be better though imho.

>Shouldn't we apply the filter after the modifications rather than before?

I don't think so, since the values of `$dr`, `$dt`, and `$ut` depend on the values in `$tr`.